### PR TITLE
Url to path with names

### PR DESCRIPTION
**Pull request by motu55**
named all urls and tried to use them everywhere
please check if names ae ok and if everythinf is there
so that we in the future can change the urls without looking at all the code

### CODE REVIEW

File: juntagrico/urls.py [hunk 1]
```diff
@@ -10,65 +9,50 @@
...
+    path('my/confirm/<str:hash>/', juntagrico_subscription.confirm, name='confirm'),
+    path('my/subscription/activate/<int:subscription_id>/', juntagrico_subscription.activate_subscription, name='sub-activate'),
+    path('my/subscription/deactivate/<int:subscription_id>/', juntagrico_subscription.deactivate_subscription, name='sub-deactivate'),
+    path('my/subscription/cancel/<int:subscription_id>/', juntagrico_subscription.cancel_subscription, name='sub-cancel'),
+    path('my/type/change/<int:subscription_id>/', juntagrico_subscription.activate_future_types, name='activate-future-types'),
+    path('my/extra/activate/<int:extra_id>/', juntagrico_subscription.activate_extra, name='extra-activate'),
+    path('my/extra/deactivate/<int:extra_id>/', juntagrico_subscription.deactivate_extra, name='extra-deactivate'),
+    path('my/extra/cancel/<int:extra_id>/<int:subscription_id>/', juntagrico_subscription.cancel_extra, name='extra-cancel'),
+    path('my/order/share/', juntagrico_subscription.order_shares, name='share-order'),
+    path('my/order/share/success', juntagrico_subscription.order_shares_success, name='share-order-succ'),
```
Comment (pascalfree) [inline juntagrico/urls.py]: I suggest `shared-order-success` or `share-ordered`

File: juntagrico/urls.py [hunk 2]
```diff
@@ -79,38 +63,36 @@
...
+    path('my/mails/send/area', juntagrico_admin.send_email_area, name='mail-area-send'),
+    path('my/mails/send', juntagrico_admin.send_email, name='mail-send'),
+    path('my/mails/send/result/<int:numsent>/', juntagrico_admin.send_email_result, name='mail-result'),
+    path('my/mails', juntagrico_admin.mails, name='mail'),
+    path('my/mails/depot', juntagrico_admin.mails_depot, name='mail-depot'),
+    path('my/mails/area', juntagrico_admin.mails_area, name='mail-area'),
+    path('my/filters', juntagrico_admin.filters, name='filters'),
+    path('my/filters/depot/<int:depot_id>/', juntagrico_admin.filters_depot, name='filter-depot'),
+    path('my/filters/area/<int:area_id>/', juntagrico_admin.filters_area, name='filter-area'),
+    path('my/subscriptions', juntagrico_admin.subscriptions, name='subs'),
```
Comment (pascalfree) [inline juntagrico/urls.py]: `filter-subs`

File: juntagrico/urls.py [hunk 3]
```diff
@@ -79,38 +63,36 @@
...
+    path('my/subscriptions/depot/<int:depot_id>/', juntagrico_admin.filter_subscriptions_depot, name='subs-depot'),
```
Comment (pascalfree) [inline juntagrico/urls.py]: `filter-subs-depot`

File: juntagrico/urls.py [hunk 4]
```diff
@@ -79,38 +63,36 @@
...
+    path('my/future', juntagrico_admin.future, name='future'),
+    path('my/export', juntagrico_admin.export, name='export'),
+    path('my/export/membersfilter', juntagrico_admin.excel_export_members_filter, name='export-membersfilter'),
+    path('my/export/members', juntagrico_admin.excel_export_members, name='export-members'),
+    path('my/export/shares', juntagrico_admin.excel_export_shares, name='export-shares'),
+    path('my/mailtemplate/<int:template_id>/', juntagrico_admin.get_mail_template, name='mail-template'),
+    path('my/waitinglist', juntagrico_admin.waitinglist, name='sub-waitinglist'),
```
Comment (pascalfree) [inline juntagrico/urls.py]: `sub-mgmt-waitinglist`

File: juntagrico/urls.py [hunk 5]
```diff
@@ -79,38 +63,36 @@
...
+    path('my/canceledlist', juntagrico_admin.canceledlist, name='sub-canceledlist'),
```
Comment (pascalfree) [inline juntagrico/urls.py]: `sub-mgmt-canceledlist`

File: juntagrico/urls.py [hunk 6]
```diff
@@ -79,38 +63,36 @@
...
+    path('my/typechangedlist', juntagrico_admin.typechangelist, name='sub-type-changelist'),
```
Comment (pascalfree) [inline juntagrico/urls.py]: `sub-mgmt-changelist`

File: juntagrico/urls.py [hunk 7]
```diff
@@ -79,38 +63,36 @@
...
+    path('my/extra/waitinglist', juntagrico_admin.extra_waitinglist, name='extra-waitinglist'),
```
Comment (pascalfree) [inline juntagrico/urls.py]: `sub-mgmt-extra-waitinglist` and the following

File: juntagrico/urls.py [hunk 8]
```diff
@@ -79,38 +63,36 @@
...
+    path('my/extra/canceledlist', juntagrico_admin.extra_canceledlist, name='extra-canceledlist'),
+    path('my/share/canceledlist', juntagrico_admin.share_canceledlist, name='share-canceledlist'),
+    path('my/pdf/depotlist', juntagrico_admin.depotlist, name='depotlist'),
```
Comment (pascalfree) [inline juntagrico/urls.py]: Maybe prefix this and following with `lists-`.

File: juntagrico/urls.py [hunk 9]
```diff
@@ -79,38 +63,36 @@
...
+    path('my/export/membersfilter', juntagrico_admin.excel_export_members_filter, name='export-membersfilter'),
+    path('my/export/members', juntagrico_admin.excel_export_members, name='export-members'),
+    path('my/export/shares', juntagrico_admin.excel_export_shares, name='export-shares'),
+    path('my/mailtemplate/<int:template_id>/', juntagrico_admin.get_mail_template, name='mail-template'),
+    path('my/waitinglist', juntagrico_admin.waitinglist, name='sub-waitinglist'),
+    path('my/canceledlist', juntagrico_admin.canceledlist, name='sub-canceledlist'),
+    path('my/typechangedlist', juntagrico_admin.typechangelist, name='sub-type-changelist'),
+    path('my/extra/waitinglist', juntagrico_admin.extra_waitinglist, name='extra-waitinglist'),
+    path('my/extra/canceledlist', juntagrico_admin.extra_canceledlist, name='extra-canceledlist'),
+    path('my/share/canceledlist', juntagrico_admin.share_canceledlist, name='share-canceledlist'),
```
Comment (pascalfree) [inline juntagrico/urls.py]: `share-mgmt-canceledlist`

File: juntagrico/urls.py (hunk 1 above)
Comment (motu55) [reply juntagrico/urls.py]: done all

**Pull request merged**